### PR TITLE
new-exporter - Use Instracluster exporter

### PR DESCRIPTION
**Pull request by cscetbon**

### CODE REVIEW

File: files/run.sh [hunk 1]
```diff
@@ -58,16 +58,17 @@ else
...
             firstNode=false
             break
         fi
     done
+
     if [ "$firstNode" = true ]; then
+        CASSANDRA_SEEDS=$(hostname -f);
         echo "***"
-        echo "Can't connect first seed, must be first node we sed CASSANDRA_SEED=$(hostname -f)" ;
+        echo "Can't connect first seed, must be first node we sed CASSANDRA_SEED=${CASSANDRA_SEEDS}" ;
```
Comment (allamand) [inline files/run.sh]: we sed -> we set

**Comment by allamand** [comment]
@cscetbon you may have forgot this file: `mv: cannot stat '/exporter.conf': No such file or directory`

**Comment by cscetbon** [comment]
@allamand yeah I did. Just fixed it. I created an issue on the project of the exporter cause it seems it prevents it from catching SIGINT. In order to change the exporter configuration it can be added to the ConfigMap as the file `exporter.conf` is also put in `/etc/cassandra`

**Review by allamand — APPROVED** [review]

**Comment by allamand** [comment]
@scetbon, do we need to wait ?

File: files/run.sh (hunk 1 above)
Comment (cscetbon) [reply files/run.sh]: fixed

**Comment by cscetbon** [comment]
@allamand whenever you're ready I am

**Comment by allamand** [comment]
There is a problem preventing the rolling update to be done, the pod stay blocked when shuting down, cf https://github.com/instaclustr/cassandra-exporter/issues/30

**Pull request merged**